src/main/java/cashleh/transaction/Expense.java

```diff
@@ -0,0 +1,11 @@
+package cashleh.transaction;
+public class Expense extends Transaction {
+    public Expense(String description, double amount) {
+        super(description, amount);
+    }
+
+    public String toString() {
+        return getDescription() + " (amount: " + getAmount() + ")";
```

Comment:
Could maybe show the amount first followed by the description and also add a "-" before the amount so that the user immediately sees that he/she is indeed shown the expenses.
e.g. "- $1,200: Monthly Rent for October"